src/lib/components/collections/req-res-section/sub-components/sub-components-header/ApiSendRequestPage.ViewModel.ts

```diff
@@ -248,9 +248,9 @@ class ApiSendRequestViewModel {
 
   public decodeRestApiData(request: any, environmentVariables): string[] {
     return [
-      this.setEnvironmentVariables(
-        this.extractURl(request.url, request),
-        environmentVariables,
+      this.extractURl(
+        this.setEnvironmentVariables(request.url, environmentVariables),
+        request,
       ),
       this.extractMethod(request.method),
       this.setEnvironmentVariables(
```

Comment:
Is env variable working on query params also? 

if I add ?asif={{asif}}

Reply:
<img width="1440" alt="Screenshot 2024-02-14 at 04 30 16" src="https://github.com/sparrowapp-dev/sparrow-app/assets/51285263/4b0439ba-e4c1-4edb-bd33-c983350f7374">

This is done. @itsmdasifraza